pages/search.vue

```diff
@@ -162,24 +172,28 @@
  ///await $fetch
 
  const updateContact = () => {
-   axios.put(`http://localhost:5000/contact/${selectedContact.id}`, selectedContact)
+  const contact = selectedContact?.value as unknown as {id: string} | undefined;
+  if (contact) {
+   axios.put(`http://localhost:5000/contact/${contact.id}`, contact)
```

Comment:
Async/await not .then

Reply:
also we should just be using the base fetch call, we don't need to be adding axios as a dependency in this project